awx/ui/src/screens/Project/Project.test.js

```diff
@@ -63,7 +63,7 @@ describe('<Project />', () => {
       '.pf-c-tabs__item-text',
       (el) => el.length === 6
     );
-    expect(tabs.at(3).text()).toEqual('Notifications');
+    expect(tabs.at(3).text()).toEqual('Job Templates');
```

Comment:
This test checks that the notifications tab is shown for admins. Since the Job Templates tab bumped Notifications back a spot, I'd change this to check that it is the fourth tab.